tools/topology/topology1/m4/google_ctc_audio_processing_coef_default.m4

```diff
@@ -0,0 +1,10 @@
+CONTROLBYTES_PRIV(DEF_CTC_PRIV,
+`       bytes "0x53,0x4f,0x46,0x34,0x00,0x00,0x00,0x00,'
```

Comment:
Navi is using IPC3 by now, the expected ABI MAGIC is
`0x53,0x4f,0x46,0x00`
rather than
`0x53,0x4f,0x46,0x34`